content/id/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -0,0 +1,333 @@
+
+    ```shell
+    kubectl create secret docker-registry myregistrykey --docker-server=DUMMY_SERVER \
+            --docker-username=DUMMY_USERNAME --docker-password=DUMMY_DOCKER_PASSWORD \
+            --docker-email=DUMMY_DOCKER_EMAIL
+    ```
+
+- Memastikan bahwa _secret_ telah terbuat.
+   ```shell
+   kubectl get secrets myregistrykey
+   ```
+
+    Keluarannya akan serupa dengan:
+
+    ```
+    NAME             TYPE                              DATA    AGE
+    myregistrykey    kubernetes.io/.dockerconfigjson   1       1d
+    ```
+
+### Menambahkan imagePullSecret ke service account
+
+Selanjutnya, modifikasi _service account_ standar dari namespace untuk menggunakan _secret_ ini sebagai imagePullSecret.
+
+
+```shell
+kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "myregistrykey"}]}'
+```
+
+Sebagai gantinya kamu dapat menggunakan `kubectl edit`, atau melakukan pengubahan secara manual _manifest_ YAML seperti di bawah ini:
+
+```shell
+kubectl get serviceaccounts default -o yaml > ./sa.yaml
+```
+
+Keluaran dari _file_ `sa.yaml` akan serupa dengan:
+
+```shell
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: 2015-08-07T22:02:39Z
+  name: default
+  namespace: default
+  resourceVersion: "243024"
+  uid: 052fb0f4-3d50-11e5-b066-42010af0d7b6
+secrets:
+- name: default-token-uudge
+```
+
+Menggunakan _editor_ pilihanmu (misalnya `vi`), buka _file_ `sa.yaml`, hapus baris dengan _key_ `resourceVersion`, tambahkan baris dengan `imagePullSecrets:` dan simpan.
+
+Keluaran dari _file_ `sa.yaml` akan serupa dengan:
+
+```shell
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: 2015-08-07T22:02:39Z
+  name: default
+  namespace: default
+  uid: 052fb0f4-3d50-11e5-b066-42010af0d7b6
+secrets:
+- name: default-token-uudge
+imagePullSecrets:
+- name: myregistrykey
+```
+
+Terakhir ganti serviceaccount dengan _file_ `sa.yaml` yang telah diperbarui.
+
+```shell
+kubectl replace serviceaccount default -f ./sa.yaml
+```
+
+### Memverifikasi imagePullSecrets sudah ditambahkan ke spesifikasi pod
+
+Ketika Pod baru dibuat dalam namespace yang sedang aktif dan menggunakan ServiceAccount, Pod baru akan memiliki _field_ `spec.imagePullSecrets` yang ditentukan secara otomatis:
+
+```shell
+kubectl run nginx --image=nginx --restart=Never
+kubectl get pod nginx -o=jsonpath='{.spec.imagePullSecrets[0].name}{"\n"}'
+```
+
+Keluarannya adalah:
+
+```
+myregistrykey
+```
+
+<!--## Menambahkan Secrets ke sebuah service account.
+
+TODO: Tes dan jelaskan bagaimana cara menambahkan secret tambahan non-K8s dengan service account yang sudah ada.
+-->
+
+## Service Account Token Volume Projection
+
+{{< feature-state for_k8s_version="v1.12" state="beta" >}}
+
+{{< note >}}
+ServiceAccountTokenVolumeProjection masih dalam tahap __beta__ untuk versi 1.12 dan diaktifkan dengan memberikan _flag_ berikut ini ke API _server_:
```

Comment:
server is indonesian word, no need to italicize https://kbbi.kemdikbud.go.id/entri/server
```suggestion
ServiceAccountTokenVolumeProjection masih dalam tahap __beta__ untuk versi 1.12 dan diaktifkan dengan memberikan _flag_ berikut ini ke API server:
```